cmd/kube-httpcache/internal/flags.go

```diff
@@ -64,7 +64,7 @@ func (f *KubeHTTPProxyFlags) Parse() error {
 	flag.StringVar(&f.Kubernetes.RetryBackoffString, "retry-backoff", "30s", "backoff for Kubernetes API reconnection attempts")
 
 	flag.StringVar(&f.Frontend.Address, "frontend-addr", "0.0.0.0", "TCP address to listen on")
-	flag.IntVar(&f.Frontend.Port, "frontend-port", 80, "TCP port to listen on")
+	flag.IntVar(&f.Frontend.Port, "frontend-port", 8080, "TCP port to listen on")
```

Comment:
I noticed that the `-frontend-port` argument is never set anywhere in the Helm templates. We could make this change possibly a bit less breaking by adding a `-frontend-port={{ .Values.service.target }}` argument to the deployment/statefulset.

Reply:
Indeed, good idea ! Amended accordingly 🙂